corehq/ex-submodules/pillowtop/tests/test_elasticsearch.py

```diff
@@ -195,6 +205,17 @@ def test_replace_doc(self):
         del doc['property']
         self._send_to_es_and_check(doc, update=True)
 
+    def test_merge_doc(self):
+        doc = {'_id': uuid.uuid4().hex, 'doc_type': 'MyCoolDoc', 'property': 'bar'}
+        self._send_to_es_and_check(doc)
+
+        update = doc.copy()
+        del update['property']
+        update['new_prop'] = 'new_val'
+        self._send_to_es_and_check(update, update=True)
+        # merging should still keep old 'property'
+        self._send_to_es_and_check(update, update=True, es_merge_update=True)
```

Comment:
I did see that but I still don't think this test is correct since it's calling `_send_to_es_and_check` with the same doc that it's using to check against.

I guess if you just got rid of the call to `_send_to_es_and_check` on line 215 then it would be OK.